packages/server-wallet/src/db/migrations/20201201095501_ledger_proposals.ts

```diff
@@ -1,27 +1,25 @@
 import * as Knex from 'knex';
 
 export async function up(knex: Knex): Promise<any> {
-  await knex.schema.table('channels', function(table) {
-    table.jsonb('my_ledger_proposal');
-    table.jsonb('their_ledger_proposal');
-    table
-      .integer('my_ledger_proposal_nonce')
-      .notNullable()
-      .defaultTo(0)
-      .unsigned();
+  await knex.schema.createTable('ledger_proposals', table => {
+    table.string('channel_id').notNullable();
```

Comment:
Should this be the table's primary key?

Reply:
I don't think so because there will be `numParticipants` number of rows

Reply:
Right. I guess `['channel_id', 'signing_address']` might be a natural private key.